### PR TITLE
feat: show a user's verification status next to its name [WPB-765]

### PR DESCRIPTION
**Pull request by caldrian**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-765" title="WPB-765" target="_blank">StoryWPB-765</a>  [iOS] 5.3.16 Indicate verified user
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

In the designs the verification status on a user profile is shown next to its name, however, currently the status was shown in the navigation bar.

This PR moves the verification information next to a user's name label.

Screenshot 2024-02-19 at 10 33 41 AM

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: wire-ios-mono.xcworkspace/contents.xcworkspacedata [hunk 1]
```diff
@@ -7,66 +7,6 @@
    <FileRef
       location = "group:README.md">
    </FileRef>
-   <FileRef
```
Comment (netbe) [inline wire-ios-mono.xcworkspace/contents.xcworkspacedata]: to be reverted, keep the files accessible to edit in Xcode

File: wire-ios/Wire-iOS Tests/ReferenceImages/ProfileDetailsViewControllerTests/test_Group_BlockingConnectionRequest_MissingLegalHoldConsent1.1.png
Comment (netbe) [inline wire-ios/Wire-iOS Tests/ReferenceImages/ProfileDetailsViewControllerTests/test_Group_BlockingConnectionRequest_MissingLegalHoldConsent1.1.png:1]: link is not red anymore, wanted?

**Review by johnxnguyen — APPROVED** [review]
Looks good 👍🏻

File: wire-ios/Wire-iOS Tests/ReferenceImages/ProfileViewControllerTests/testForWrapInNavigationController.1.png
Comment (netbe) [inline wire-ios/Wire-iOS Tests/ReferenceImages/ProfileViewControllerTests/testForWrapInNavigationController.1.png:1]: this is weird here, no? we don't know who's profile we're seeing

File: wire-ios/Wire-iOS Tests/ReferenceImages/ProfileViewControllerTests/testForDeviceListContext.1.png
Comment (netbe) [inline wire-ios/Wire-iOS Tests/ReferenceImages/ProfileViewControllerTests/testForDeviceListContext.1.png:1]: same goes here, no info which profile we're seeing, is it expected?

**Review by netbe — CHANGES_REQUESTED** [review]
see my comments, but essentially there could be an issue with the view for devices which show no profile info now

File: wire-ios/Wire-iOS Tests/ReferenceImages/ProfileDetailsViewControllerTests/test_Group_BlockingConnectionRequest_MissingLegalHoldConsent1.1.png
Comment (caldrian) [reply wire-ios/Wire-iOS Tests/ReferenceImages/ProfileDetailsViewControllerTests/test_Group_BlockingConnectionRequest_MissingLegalHoldConsent1.1.png:1]: According to @agisilaos this is correct now. The link texts should appear in accent color (here blue).

File: wire-ios/Wire-iOS Tests/ReferenceImages/ProfileViewControllerTests/testForDeviceListContext.1.png
Comment (caldrian) [reply wire-ios/Wire-iOS Tests/ReferenceImages/ProfileViewControllerTests/testForDeviceListContext.1.png:1]: No, according to the designs, it is correct.
Screenshot 2024-02-19 at 2 41 35 PM

File: wire-ios/Wire-iOS Tests/ReferenceImages/ProfileViewControllerTests/testForWrapInNavigationController.1.png
Comment (caldrian) [reply wire-ios/Wire-iOS Tests/ReferenceImages/ProfileViewControllerTests/testForWrapInNavigationController.1.png:1]: It should be correct like this, according to the design.

**Review by agisilaos — APPROVED** [review]

**Review by netbe — APPROVED** [review]

**Pull request merged**